public/data/eddiejaoude.json

```
@@ -43,6 +43,16 @@
       "name": "Listen to my Podcast",
       "url": "https://anchor.fm/eddiejaoude",
       "icon": "codeforces"
+    },
+    {
+      "name": "Donate to me on GitHub Sponsors",
+      "url": "https://github.com/sponsors/eddiejaoude",
+      "icon": "dollar"
+    },
+    {
+      "name": "Customising your GitHub Profile course (free)",
+      "url": "http://eddiejaoude.io/course-github-profile-landing",
```

It's using the secure extension, so I believe we should update it!! 
```suggestion
      "url": "https://eddiejaoude.io/course-github-profile-landing",
```

---

Thank you, but the link is without https and it redirects to the https on a sub domain